site/en/docs/extensions/mv3/known-issues/index.md

```diff
@@ -57,11 +49,21 @@ This section lists major features that will be added to the Manifest V3 platform
 
 **Tracking issue:** [1247690][]
 
-### Service workers are not started in response to webRequest events
+## Fixed but not yet in Stable {: #resolved }
+
+### webRequest.onAuthRequired events {: #webrequest-onauthrequired }
+
+**Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.
+
+**Estimated timeline:** Slated for release in Chrome 108.
+
+**Tracking issue:** [1135492][]
```

Comment:
```suggestion
```

Reply:
If it's shipped, I don't think anyone will look at the bug.

Reply:
Ack.